parsl/configs/cori_ipp_multinode.py

```diff
@@ -36,7 +36,8 @@
                 tasks_per_node=2,
                 init_blocks=1,
                 max_blocks=1,
-                overrides='OVERRIDES',     # Please replace OVERRIDES with your overrides
+                scheduler_options='SCHEDULER_OPTIONS',     # Please replace SCHEDULER_OPTIONS with your scheduler_options
+                worker_init='WORKER_INIT',     # Please replace WORKER_INIT with your worker_init
                 launcher=SrunLauncher,
```

Comment:
I know this isn't part of your PR, but `SrunLauncher` should be `SrunLauncher()`, no?